### DISCUSSION TITLE
group-hover has precedence over group-focus-within

### DISCUSSION THREAD

**Discussion by glecetre** [tailwindlabs/tailwindcss#11019] [Help] [answered]
Hello, 

I see that when using `group-hover` and `group-focus-within` to style a whole group's hover and focus state, the focus state is actually always replaced by the hover state, as long as the mouse is hovering. I would have expected that the focus has more precedence than hovering.

Here's a reproduction of what I mean: https://play.tailwindcss.com/MtpxrGqK8W.

In this example, I would like the input to keep its red background even when hovering the group, given that it's focused.
How should I achieve that?

Thanks for your help :)

**Comment by wongjn** [comment, marked as answer]
You could use a class with stacked modifiers with both `group-hover` and `group-focus-within` to explicitly declare the `background-color` when `:focus-within` and `:hover` pseudo classes match. Alternatively, you could use the [important modifier `!`](https://tailwindcss.com/docs/configuration#important-modifier) to ensure the `group-focus-within` always takes precedence. See both solutions in [this Tailwind Play](https://play.tailwindcss.com/d6lk4RhYig).

**Reply by glecetre** [reply]
How did I not think of that... Thanks :)
